gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/GradeImportUploadStep.java

```diff
@@ -99,20 +99,20 @@ public void onSubmit() {
 				try {
 					spreadsheetWrapper = ImportGradesHelper.parseImportedGradeFile(upload.getInputStream(), upload.getContentType(), upload.getClientFileName(), userMap);
 				} catch (final GbImportExportInvalidColumnException e) {
-					log.debug("GBNG import error", e);
-					error(getString("importExport.error.incorrectformat"));
+					log.debug("incorrect format", e);
+					error(getString("importExport.error.incorrectformat")+" - "+e.getMessage());
 					return;
 				} catch (final GbImportExportInvalidFileTypeException | InvalidFormatException e) {
-					log.debug("GBNG import error", e);
-					error(getString("importExport.error.incorrecttype"));
+					log.debug("incorrect type", e);
+					error(getString("importExport.error.incorrecttype")+" - "+e.getMessage());
 					return;
 				} catch (final GbImportExportDuplicateColumnException e) {
-					log.debug("GBNG import error", e);
-					error(getString("importExport.error.duplicatecolumn"));
+					log.debug("duplicate column", e);
+					error(getString("importExport.error.duplicatecolumn")+" - "+e.getMessage());
```

Comment:
I haven't actually reverted it, I clicked the button but didn't proceed.
The line that is removed was localised:
`error(getString("importExport.error.duplicatecolumn"));`

The line that was added is not:
`error(getString("importExport.error.duplicatecolumn")+" - "+e.getMessage());`

the `error(some string)` method in Wicket puts the message into the FeedbackPanel stack which is then displayed to the user immediately.